tests/tx/test_indexes.py

```diff
@@ -68,13 +70,14 @@ def test_tx_tips_with_conflict(self):
         self.assertNotEqual(tx2.hash, tx3.hash)
         self.assertTrue(self.manager.propagate_tx(tx3, False))
         self.assertIn(tx3.hash, tx2.get_metadata().conflict_with)
-        self.assertEqual(
-            {tx.hash for tx in self.manager.tx_storage.indexes.mempool_tips.iter(self.manager.tx_storage)},
-            # XXX: what should we expect here? I don't think we should exclude both tx2 and tx3, but maybe let the
-            # function using the index decide
-            # {tx1.hash, tx3.hash}
-            {tx1.hash}
-        )
+        if self.manager._enable_sync_v2:
```

Comment:
Isn't it better to check if the index itself exists (instead of sync v2)? It seems safer for future changes.

Reply:
Thanks!